data/tests/fwupdtool.sh

```diff
@@ -230,6 +230,26 @@ if [ -d $BASEDIR ]; then
        rc=$?; if [ $rc != 3 ]; then error $rc; fi
 fi
 
+if [ -x /usr/bin/certtool ]; then
```

Comment:
Which ones are missing? I wonder if this should be a $CI || [ -x ]

Reply:
it failed on the snap tests -- and I kinda figured it was a pretty heavy ask to runtime depend on certtool just to run the installed tests.

Reply:
Ah got it. We probably shouldn't add it to the snap. Sounds good.